pkg/oauth/apis/oauth/validation/validation.go

```diff
@@ -79,8 +94,24 @@ func ValidateAccessToken(accessToken *oauthapi.OAuthAccessToken) field.ErrorList
 
 func ValidateAccessTokenUpdate(newToken, oldToken *oauthapi.OAuthAccessToken) field.ErrorList {
 	allErrs := validation.ValidateObjectMetaUpdate(&newToken.ObjectMeta, &oldToken.ObjectMeta, field.NewPath("metadata"))
+	// since TimeoutsIn can be concurrently updated by multipe masters we do
```

Comment:
We should error if `oldToken.TimeoutsIn == 0 && newToken.TimeoutsIn != 0` to prevent a token that was never supposed to timeout from being updated to timeout.  Probably needs a comment describing how this race could happen between multiple masters.

Reply:
yes